tools/Utilities/src/Strings/en-us/Resources.resw

```diff
@@ -118,14 +118,13 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="EnvVariablesEditorUtilityDesc" xml:space="preserve">
-    <value>A quick utility for managing environment variables.</value>
+    <value>Create and manage variable sets with a lightweight UI.</value>
   </data>
   <data name="EnvVariablesEditorUtilityTitle" xml:space="preserve">
     <value>Environment Variables Editor</value>
   </data>
   <data name="HostsFileEditorUtilityDesc" xml:space="preserve">
-    <value>Quick and simple utility for managing hosts file.</value>
-    <comment>;Locked={"hosts"}</comment>
+    <value>Add, filter, or remove file entries in less time.</value>

```

Comment:
Compared to what?

Reply:
Notepad, other text editors I believe..

Reply:
"Add, filter or remove Hosts file entries" ?

cc: @cinnamon-msft

Reply:
updated